File: Challenge 1/c1easy.pl
#!/usr/bin/perl
use warnings;
use strict;

open(OUTPUT, "> perlOut.txt") or die "Could not open file!";

print "Please enter your name: ";
chomp( my $name = <>);

print "Please enter your age: ";
chomp( my $age = <>);

print "Please enter your username: ";
chomp( my $uname = <>);

select OUTPUT;
print "Your name is $name, you are $age years old, and your username is $uname.";

File: Challenge 1/c1easy.py
name = input("Please enter your name: ")
age = input("Please enter your age: ")
uname = input("Please enter your username: ")

outFile = open("pythonOut.txt", 'w')
outFile.write("Your name is " +name +", you are " +age +" years old, and your username is " +uname)
outFile.close()
